bin/do_xcf.py

```diff
@@ -182,11 +182,13 @@ def corr_func(p):
     rps=cfs[:,2,:]
     rts=cfs[:,3,:]
     zs=cfs[:,4,:]
+    nbs=cfs[:,5,:]
     cfs=cfs[:,1,:]
 
     rp = (rps*wes).sum(axis=0)/wes.sum(axis=0)
     rt = (rts*wes).sum(axis=0)/wes.sum(axis=0)
     z = (zs*wes).sum(axis=0)/wes.sum(axis=0)
+    nb = (nbs*wes).sum(axis=0)/wes.sum(axis=0)
```

Comment:
remove the weights, the number of pairs should be an intenger. Then nb = nbs.sum(axis=0)